pkg/installation/util.go

```diff
@@ -76,7 +76,12 @@ func getDownloadTarget(index index.Plugin) (version, uri string, fos []index.Fil
 	version, uri = getPluginVersion(p)
 	glog.V(4).Infof("Matching plugin version is %s", version)
 
-	return version, uri, p.Files, p.Bin, nil
+	files := p.Files
+	if len(files) == 0 {
+		files = []index.FileOperation{{From: "*", To: "."}}
```

Comment:
worth adding a glog.V(4) here saying `file operation not specified, assuming %v`